src/test_driver/linux-cirque/README.md

```diff
@@ -53,6 +53,10 @@ logs).
 > Note: The container (as well as the networks) will be preserved for debugging.
 > You may need to clean it by yourself.
 
+> Building objects (e.g. chip-lighting-app, chip-echo-responder, etc.) are
+> required in the suitable directories. (Run [gn_build.sh](../../../gn_build.sh)
```

Comment:
Could you define 'suitable directories' and where this is configured/needs  changing?

I am looking to move builds generally using `scripts/build/build_example.py` and the directories that generates are likely different than gn_build.  Would need to understand what to change to be able to use generic paths.

Reply:
@andy31415 Thanks fo the review.
I wrote vaguely since I thought it might be possible to use a different folder for the new test file.
Test files in linux-cirque use building objects in `out/debug/linux_x64_gcc/` or `out/debug/standalone/` folders like below at the moment.
- EchoTest.py : [CHIP_REPO/out/debug/linux_x64_gcc/chip-echo-responder](https://github.com/project-chip/connectedhomeip/blob/2937f1ae79909848463229e525845ca427ab06bc/src/test_driver/linux-cirque/EchoTest.py#L85)
- EchoOverTcpTest.py : [CHIP_REPO/out/debug/linux_x64_gcc/chip-echo-responder](https://github.com/project-chip/connectedhomeip/blob/b9f6c60d8026381c1ed9b79a5569e4e37a1084a0/src/test_driver/linux-cirque/EchoOverTcpTest.py#L85), [CHIP_REPO/out/debug/linux_x64_gcc/chip-echo-requester](https://github.com/project-chip/connectedhomeip/blob/b9f6c60d8026381c1ed9b79a5569e4e37a1084a0/src/test_driver/linux-cirque/EchoOverTcpTest.py#L89)
- InteractionModelTest.py : [CHIP_REPO/out/debug/linux_x64_gcc/chip-im-responder](https://github.com/project-chip/connectedhomeip/blob/b9f6c60d8026381c1ed9b79a5569e4e37a1084a0/src/test_driver/linux-cirque/InteractionModelTest.py#L85), [CHIP_REPO, "out/debug/linux_x64_gcc/chip-im-initiator](https://github.com/project-chip/connectedhomeip/blob/b9f6c60d8026381c1ed9b79a5569e4e37a1084a0/src/test_driver/linux-cirque/InteractionModelTest.py#L89)
- MobileDeviceTest.py : [CHIP_REPO/out/debug/standalone/chip-lighting-app](https://github.com/project-chip/connectedhomeip/blob/b9f6c60d8026381c1ed9b79a5569e4e37a1084a0/src/test_driver/linux-cirque/MobileDeviceTest.py#L83)
- OnOffClusterTest.py : [CHIP_REPO/out/debug/standalone/chip-lighting-app](https://github.com/project-chip/connectedhomeip/blob/b9f6c60d8026381c1ed9b79a5569e4e37a1084a0/src/test_driver/linux-cirque/OnOffClusterTest.py#L85), [CHIPREPO/out/debug/standalone/chip-tool](https://github.com/project-chip/connectedhomeip/blob/b9f6c60d8026381c1ed9b79a5569e4e37a1084a0/src/test_driver/linux-cirque/OnOffClusterTest.py#L89)

I mentioned `gn_build.sh` since the [cirque workflow](https://github.com/project-chip/connectedhomeip/blob/b9f6c60d8026381c1ed9b79a5569e4e37a1084a0/.github/workflows/cirque.yaml#L101) uses it. I know this sentence isn't perfect, but I wrote it to prevent first-timers from getting lost. Would it be better to add this later instead of now?

Reply:
Some comment of this form may help:

```md
> Binary paths are hard-coded in the relevant unit tests, generally 
> with code of the form  `os.path.join(CHIP_REPO, ...`. The hardcoded paths
> follow use the binary path valid after `gn_build.sh`
```

This may help us understand what to search for (the path.join bit) if we ever
decide to udate things.

Reply:
Edited with your advice. Thank you!

Reply:
@andy31415 Please review my latest patch and resolve this conversation. Thanks!